Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
+
+extension ExtendedForReactiveness {
+	/// A proxy which exposes the reactivity of `self`.
+	public var rac: Reactive<Self> {
```

Comment:
Why not `reactive`? Consider usage:

```
// "Give me rac notifications for…"
NSNotificationCenter.default.rac.notifications(forName:

// "Give me rac data with…"
someURLSession.rac.data(with:

// "Give me reactive notifications for…"
NSNotificationCenter.default.reactive.notifications(forName:

// "Give me reactive data with…"
someURLSession.reactive.data(with:
```

The latter feels more readable/sensical to me.